pbm/bsontypes.go

```diff
@@ -11,6 +11,10 @@ type OpTime struct {
 	Term int64               `bson:"t" json:"t"`
 }
 
+type Tags struct {
+    Name map[string] string `bson:"omitempty" json:"omitempty"`
+}
+
```

Comment:
I don't see any usage of this type

Reply:
It's used into NodeInfo Struct; it collects the Tag array output for dbAdminCommand sent

Reply:
In node NodeInfo struct you use `map[string]string` as type (https://github.com/percona/percona-backup-mongodb/blob/751ea12e9ae9753843c715c3cf05ab1d26a1d5a0/pbm/bsontypes.go#L40)

Reply:
You are right, it's redundant, i will remove Struct

Reply:
Struct has been removed